### ISSUE TITLE
deepgemm_fp8_paged_mqa_logits crashes with SIGFPE for ChunkK=64 on gfx950

### ISSUE THREAD

**Issue by stefanskiasan** [ROCm/aiter#5298]
**`deepgemm_fp8_paged_mqa_logits` crashes with SIGFPE for `ChunkK=64`**

On gfx950, calling the Gluon paged-MQA-logits kernel with `ChunkK=64` and
`KVBlockSize=16` terminates the process with `Floating point exception (core
dumped)`. The wrapper's own precondition accepts the combination:

```python
assert ChunkK % KVBlockSize == 0 or KVBlockSize % ChunkK == 0
# 64 % 16 == 0 -> passes
```

**Reproduction** (aiter 0.1.21.post1, ROCm 7.2.3, MI355X / gfx950):

```python
import torch
from aiter.ops.triton.attention.pa_mqa_logits import deepgemm_fp8_paged_mqa_logits

BLOCK, HEADS, DIM, CTX = 16, 32, 128, 131072
batch, next_n = 8, 5
nb = (CTX + BLOCK - 1) // BLOCK
q  = torch.randn(batch, next_n, HEADS, DIM, device="cuda", dtype=torch.bfloat16).to(torch.float8_e4m3fn)
kv = torch.randn(nb, BLOCK, 1, DIM + 4, device="cuda", dtype=torch.bfloat16).to(torch.uint8)
w  = torch.randn(batch * next_n, HEADS, device="cuda", dtype=torch.float32)
cl = torch.full((batch,), CTX, device="cuda", dtype=torch.int32)
bt = torch.arange(nb, device="cuda", dtype=torch.int32).repeat(batch, 1)
out = torch.empty(batch * next_n, CTX, device="cuda", dtype=torch.float32)

deepgemm_fp8_paged_mqa_logits(q, kv, w, out, cl, bt, CTX,
                              Preshuffle=True, KVBlockSize=BLOCK,
                              ChunkK=64, WavePerEU=8)   # -> SIGFPE
```

`ChunkK` of 128, 256 and 512 all work with the same inputs.

**Context — why we tried it.** The call site in vLLM
(`rocm_aiter_mla_sparse.py`) hard-codes `ChunkK=256, WavePerEU=2`. Sweeping
those on GLM-5.3 (`index_n_heads=32`, `index_head_dim=128`, block size 16,
`index_topk=2048`) shows the defaults are not optimal for this shape:

| rows | ChunkK / WavePerEU | median | vs default |
|---:|---|---:|---:|
| 40 | 256 / 2 (default) | 0.0661 ms | — |
| 40 | **128 / 8** | **0.0544 ms** | **−17.7 %** |
| 320 | 256 / 2 (default) | 0.3770 ms | — |
| 320 | **128 / 8** | **0.3237 ms** | **−14.1 %** |

The trend points toward smaller `ChunkK`, which is why 64 was tried. Three
repeats per point, median reported; 8 warm-up calls; `torch.cuda.synchronize()`
around each timing window.

Two side notes from reading the wrapper:
* `WavePerEU` is overridden internally (to 4 or 1), but only under
  `if get_gfx() == "gfx1250"`. On gfx950 the caller's value stands.
* `WavePerEU=16` compiles but warns: *"failed to meet occupancy target given
  by 'amdgpu-waves-per-eu' … desired occupancy was 16, final occupancy is 8"*.

Happy to test a fix on gfx950.
